plugins/protocolbuffers/cpp/v21.7/buf.plugin.yaml

```diff
@@ -5,3 +5,4 @@ source_url: https://github.com/protocolbuffers/protobuf
 description: Official protobuf plugin for C++.
 output_languages:
   - cpp
+spdx_license_id: BSD-3-Clause
```

Comment:
this one is a bit tricky, as [its license](https://github.com/protocolbuffers/protobuf/blob/v21.7/LICENSE) is not recognized by github, so I put it into [licensecheck](https://github.com/google/licensecheck) and it suggested it is [BSD-3-Clause](https://spdx.org/licenses/BSD-3-Clause.html). it looks really similar but it will be great if someone can also have a look

Reply:
That looks right to me - https://github.com/protocolbuffers/protobuf/commit/24bf56fb3a2fab42d355b15de11782c3144b9e80 mentions a change from Apache 2 to "New BSD", which corresponds to "[BSD 3-Clause "New" or "Revised" License](https://spdx.org/licenses/BSD-3-Clause.html).